### ISSUE TITLE
@kchenery 

### ISSUE THREAD

**Issue by gyssels** [exercism/powershell#79]
Sad I don't have admin rights to add you to the project.

I liked your improvement in adding the documentation <# #> on the files.Sorry to erased then first. I add to then after the merge to master on reverse and sum-of-multiples. Do you have any others improvements that you think is fit to the project?

**Comment by kchenery** [comment]
Ive been wondering about the merging approach you're using.  I'm finding it hard to follow the flow of changes as a result.  Probably just me 😄   

I think you're doing this to ensure the config.json file is correctly updated (and also avoid conflicts to it...is that right?).  Perhaps another way would be to run some kind of integration branch alongside master that our exercises are merged into.  The integration branch then gets the config.json updated and when happy is merged into master.  Could be more work.  I dunno.  Just spit balling to be honest.

I'll go add some comment based help to the other exercises and submit a PR for that.  I think its nice to have them in the examples at least.  Its also useful in the exercise stubs - especially if we need to introduce a weird type to one of the parameters.  The examples will help the pupils understand the interfaces.

**Comment by gyssels** [comment]
**As for now (v1)** I'm using a per exercise-x-branch as a integrations branch, supposed that will be more than two there. this will sure help in merges as we are only merging exercises files.
Step 1 ) master -> exercise-x-branch: Creating the exercise integration band
Step 2 ) exercise-x-branch <- upload some exercise file to help devs implements code logic and fasten result
Step 3..n ) exercise-x-branch <- devs implements exercise
Step [last-2] ) exercise-x-branch <- control exercises, pass Checks and upload final files (config.json)
Step [last-1] ) master -> exercise-x-branch: merge and solve conflics, lasts checks
Step [last] ) master <- exercise-x-branch: save merge and publish

**Comment by gyssels** [comment]
**As for (v2)** I finished, and am polishing the auto-generation of all 100 exercises (PS-> automation tool).
I thinking creating a unique all generated exercises branch and working from there and get us modifying the example files for a real one.

**Comment by gyssels** [comment]
Feel free to implement exercises 9-20. I'm autogen exercises > 20

**Comment by gyssels** [comment]
Actually, Microsoft recommends the "comment based help" be between the function declaration and the PARAM() definition, as it travels with code in copy and paste scenarios.

Function Get-SumOfMultiples {
    <# !here! #>
    [CmdletBinding()]

**Comment by kchenery** [comment]
I think where I'm getting confused is your step [last - 2] item.  You dont appear to be merging our branches so I have hanging branches in my clone.  See: https://github.com/exercism/powershell/network for an example.

All good I can kick them to the dust.

I've got a solution for Exercise 9 (Accumulate) though I'm not happy with it.  Might raise a WIP PR and see what people think/recommend.  There's probably a better way to my thinking.

And I'll do those help pieces shortly too.

**Comment by gyssels** [comment]
Ok. I guess as I come from TFS I got to familiarize with the git vocabulary. it could be the why, but is definitively merged as it is published in main.

**Comment by gyssels** [comment]
a branch only for you ;) Exercises 9-20: Exercise-Kchenery

**Comment by gyssels** [comment]
I guess is understable the confusing of best practice in branch code. We should "Always" do a reverse integration (RI) and code testing before a foward integration (FI). In other terms: push from master, test, push to master. The symptoms your are experiencing is more about me experiencing the github commands , i promise most will disappear ;) as I'm getting the vocabs.

**Comment by gyssels** [comment]
![branchingcode](https://user-images.githubusercontent.com/13238035/41566655-b07dd1c8-735c-11e8-9944-bb14e2db0671.png)

**Comment by kchenery** [comment]
I understand branching. And I understand the need to merge from master into dev (at times) before you can merge dev into master.

What I'm not getting or seeing are the merges from the "dev" branches into master.  However; if its going to go away dont worry about it.
